test/trackpad-scroll-test.html

```diff
@@ -0,0 +1,156 @@
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            margin: 20px;
+        }
+        .test-container {
+            border: 1px solid #ccc;
+            margin: 20px 0;
+            padding: 20px;
+        }
+        .viewer {
+            width: 800px;
+            height: 600px;
+            border: 1px solid #000;
+        }
+        .info {
+            background: #f0f0f0;
+            padding: 10px;
+            margin: 10px 0;
+        }
+        .controls {
+            margin: 10px 0;
+        }
+        .controls label {
+            display: inline-block;
+            width: 200px;
+        }
+        .controls input {
+            width: 100px;
+        }
+    </style>
+</head>
+<body>
+    <h1>OpenSeadragon Trackpad Scroll Speed Test</h1>
+    
+    <div class="info">
+        <h3>Test Instructions:</h3>
+        <ol>
+            <li>Try scrolling with a mouse wheel - note the zoom speed</li>
+            <li>Try scrolling with a trackpad - note the zoom speed</li>
+            <li>Adjust the trackpad sensitivity slider and test again</li>
+            <li>The trackpad should feel more similar to mouse wheel scrolling with the fix</li>
+        </ol>
+    </div>
+
+    <div class="controls">
+        <label for="trackpadSensitivity">Trackpad Sensitivity:</label>
+        <input type="range" id="trackpadSensitivity" min="0.1" max="1.0" step="0.1" value="0.3">
+        <span id="sensitivityValue">0.3</span>
+        <button onclick="updateSensitivity()">Update</button>
+    </div>
+
+    <div class="test-container">
+        <h3>Test Viewer (with trackpad scroll fix)</h3>
+        <div id="viewer1" class="viewer"></div>
+    </div>
+
+    <div class="test-container">
+        <h3>Reference Viewer (original behavior)</h3>
+        <div id="viewer2" class="viewer"></div>
+    </div>
+
+    <div class="info">
+        <h3>Scroll Event Debug Info:</h3>
+        <div id="debugInfo">
+            <p>Scroll events will appear here...</p>
+        </div>
+    </div>
+
+    <script src="../build/openseadragon/openseadragon.js"></script>
+    <script>
+        let viewer1, viewer2;
+        let scrollEventCount = 0;
+
+        // Initialize viewers
+        function initViewers() {
+            // Viewer with trackpad scroll fix (default behavior)
+            viewer1 = OpenSeadragon({
+                id: 'viewer1',
+                prefixUrl: '../build/openseadragon/images/',
+                tileSources: {
+                    type: 'image',
+                    url: '../test/data/A.png'
+                },
+                trackpadScrollSensitivity: 0.3
+            });
+
+            // Reference viewer (simulating original behavior)
+            viewer2 = OpenSeadragon({
+                id: 'viewer2',
+                prefixUrl: '../build/openseadragon/images/',
+                tileSources: {
+                    type: 'image',
+                    url: '../test/data/A.png'
+                }
```

Comment:
This says it's simulating the original behavior, but it's not. It would need a `trackpadScrollSensitivity` of 1.